src/components/form-control/form-control.css

```diff
@@ -5,7 +5,7 @@
 .label {
   color: var(--mds-grey-54);
   display: inline-flex;
-  font: var(--mds-type-subtext);
+  font: var(--mds-type-subtext) !important;
```

Comment:
I feel kind of weird about this. I understand we want to combat pollution, and admittedly if someone really wants a different style they can use their own `!important` and/or not compose this class, I just worry about the arms race that can develop when using `!important`